doc/source/whatsnew/v0.23.0.txt

```diff
@@ -308,7 +308,10 @@ Conversion
 - Bug in :class:`DatetimeIndex` where adding or subtracting an array-like of ``DateOffset`` objects either raised (``np.array``, ``pd.Index``) or broadcast incorrectly (``pd.Series``) (:issue:`18849`)
 - Bug in :class:`Series` floor-division where operating on a scalar ``timedelta`` raises an exception (:issue:`18846`)
 - Bug in :class:`Index` constructor with ``dtype=CategoricalDtype(...)`` where ``categories`` and ``ordered`` are not maintained (issue:`19032`) 
-
+- Bug in :class:`Series`` with ``dtype='timedelta64[ns]`` where addition or subtraction of ``TimedeltaIndex`` had results cast to ``dtype='int64'`` (:issue:`17250`)
+- Bug in :class:`TimedeltaIndex` where division by a ``Series`` would return a ``TimedeltaIndex`` instead of a ``Series`` (issue:`19042`)
```

Comment:
you crossed this out is that correct?

Reply:
The original PR fixed both `TimedeltaIndex.__mul__` and `TimedeltaIndex.__div__`.  Then we narrowed the scope so it only fixed `__div__`.

Reply:
ok, so is that issue closed by this PR? (you cross it out i the header of the PR)

Reply:
Correct, that #19042 is not fully closed by this PR.

Reply:
ok, let's close #19042 with this, and pls create a new issue for the residual. (and ref this PR)